lib/rack/lint.rb

```diff
@@ -714,6 +720,28 @@ def verify_content_length(size)
         end
       end
 
+      ## 
+      ## ==== The +rack.protocol+ Header
+      ##
+      def check_rack_protocol_header(status, headers)
+        ## If the +rack.protocol+ header is present, it must be a String, and
+        ## must be one of the values from the +rack.protocols+ array from the
```

Comment:
Do we want to name the env key different than the response header?  I'm not against that, but doing so is inconsistent with the changelog and earlier changes in the diff and the code that Lint runs.

Reply:
No, I made a mistake, it should be `rack.protocol` in both directions.